docs/modules/setup/pages/third-party-tools.adoc

```diff
@@ -37,3 +45,22 @@ Add any Kroki URL after `https://niolesk.top/#` and you'll be able to edit your
 
 Here's an example: https://niolesk.top/#https://kroki.io/plantuml/svg/eNpTLMlIzU1VyM3MK6nk4nLKT9K1c8zJTE61UvBIzcnJVwQArksKZQ==
 ====
+
+== VS-Code
+
+The following {url-vscode}[Visual Studio Code] plugins support Kroki:
+
+ * {url-vscode-asciidoctor}[AsciiDoc]
+ * {url-vscode-asciidoc-slides}[AsciiDoc Slides]
+
+== Gitlab
+
+{url-gitlab}[Gitlab] can {url-gitlab-int}[use Kroki to render Images] in Markdown, AsciiDoc, reST and Textile documents.
+
+== Sphinx Documentation
+
+There is an {url-sphinx-int}[Kroki integration] for the {url-sphinx}[Sphinx documentation generator].
+
+== Commercial products
+
+* {url-conf-kroki}[Kroki Diagrams for Confluence]
```

Comment:
I wasn't aware of this commercial offering and to be honest I don't want to include it.
They (try) to make money off the public instance (https://kroki.io) which is not fair.

I wouldn't mind if they were using their own instance but using the public instance (kindly hosted by Exoscale) in a commercial product is wrong!

Reply:
No worries (I totally support you in this). I did a quick google search and found them, but didn't really look into it. Did you contact them?